cmd/kaniko-docker/main.go

```diff
@@ -363,6 +378,9 @@ func run(c *cli.Context) error {
 	username := c.String("username")
 	noPush := c.Bool("no-push")
 	configOverride := c.String("dockerconfig")
+	dockerUsername := c.String("docker-username")
```

Comment:
We can keep the variables name like `baseImageUsername` which makes it more clear. Similar for others